lib/zlib.js

```diff
@@ -0,0 +1,7 @@
+'use strict'
+module.exports = select(process.version)
+module.exports._SELECT_ZLIB = select
+
+function select (version) {
+  return version >= 'v9' ? require('zlib') : require('minizlib')
```

Comment:
```js
> 'v9.0.0' >= 'v9'
true
> 'v9.1.0' >= 'v9'
true
> 'v10.0.0' >= 'v9'
false
```

… just saying :)

Reply:
The price I pay for being maximally lazy XD